WordPress/gutenberg
# Remove hard coded color from query pagination icons

## Description


When selected in the list view, the block's icon color did not change from black to white.
This PR removes hard coded colors from the query pagination icons.
Closes https://github.com/WordPress/gutenberg/issues/31989

## How has this been tested?

1. Add a query pagination block
2. Open the list view and select the block.
3. Confirm that the icon color changes from black to white, re-test with the inner blocks.

## Screenshots 
![Icon color is white when the block is selected](https://user-images.githubusercontent.com/7422055/138296685-533de829-74db-4c91-a0ba-b046d408b6e5.png)

## Types of changes




Enhancement

## Checklist:
